src/stdune/loc.ml

```diff
@@ -103,22 +119,26 @@ let pp_file_excerpt ~context_lines ~max_lines_to_print_in_full pp
       if exists then
         pp_file_excerpt ()
       else
-        Result.Ok ()
+        Result.Ok Pp.nop
     with
+    | Ok pp -> pp
     | Error exn ->
       let backtrace = Printexc.get_backtrace () in
       Format.eprintf "Raised when trying to print location contents of %s@.%a@."
         file
         (Exn.pp_uncaught ~backtrace)
-        exn
-    | Ok () -> Format.pp_print_flush pp ()
+        exn;
+      Pp.nop
 
-let print ppf ({ start; stop } as loc) =
+let pp ({ start; stop } as loc) =
   let start_c = start.pos_cnum - start.pos_bol in
   let stop_c = stop.pos_cnum - start.pos_bol in
-  Format.fprintf ppf "@{<loc>File \"%s\", line %d, characters %d-%d:@}@\n"
-    start.pos_fname start.pos_lnum start_c stop_c;
-  pp_file_excerpt ppf ~context_lines:2 ~max_lines_to_print_in_full:10 loc
+  let open Pp.O in
+  Pp.verbatim
```

Comment:
We are missing a `Pp.tag Loc` here.

Reply:
I've added it back. Although I'm not sure what it does exactly.

Reply:
It puts the line in bold. The meaning is encoded in `User_message.Print_config.default`.